### PR TITLE
Toronto 2015 - Adding New Relic and uberflip as sponsors.

### PR DESCRIPTION
**Pull request by WhatsChrisDoing**

### CODE REVIEW

File: site/content/events/2015-toronto/_sponsors.txt [hunk 1]
```diff
@@ -54,6 +55,7 @@ filter: erb
 
 <h1>Media sponsors</h1>
 <a href="http://oreilly.com/"><img border="0" alt="O'Reilly Media" title="O'Reilly Media" width="100px" height="100px" src='<%= File.join('/',@eventhome,"logos/oreilly.png")%>'></a>
+<a href="http://uberflip.com"><img border="0" alt='Uberflip' title='Uberflip' width="100px" height="100px" src='<%= File.join('/',@eventhome,"logos/uberflip.png")%>'></a>
```
Comment (bridgetkromhout) [inline site/content/events/2015-toronto/_sponsors.txt]: `"logos/oreilly.png")%>'></a>` should be `"logos/oreilly.png")%>'></a>,`

**Comment by WhatsChrisDoing** [comment]
Bridget, I think the HTML was okay before but the new commit makes all of the sponsor handling consistent.

**Comment by bridgetkromhout** [comment]
A missing comma can cause this: https://github.com/jedi4ever/devopsdays-webby/commit/1c2fcbf4c8e923c6f29afdd692a0ea7ad95637ac

It looks good now.

**Comment by WhatsChrisDoing** [comment]
That would be bad. The new page is easy to maintain in any case. Thank you keeping all of the details covered.

**Pull request merged**